FBC/DebtAndEquities/CreditEvents.rdf

```diff
@@ -68,18 +69,24 @@
 	
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;BreachOfCovenant">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;CreditEvent"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;involves"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-agr-ctr;Contract"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">breach of covenant</rdfs:label>
 		<skos:definition xml:lang="en">credit event representing a violation of an express, or implied, condition of a contract to do or not to do something that has a negative financial impact</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;CreditEvent">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-oc;Occurrence"/>
 		<rdfs:label xml:lang="en">credit event</rdfs:label>
-		<skos:definition xml:lang="en">event signifying a sudden change in a borrower&apos;s credit standing, such as bankruptcy or a violation of a bond indenture or loan agreement, that raises doubts about the borrower&apos;s ability to meet current or future obligations</skos:definition>
+		<skos:definition xml:lang="en">event signifying a sudden change in credit standing, such as bankruptcy or a violation of a bond indenture or loan agreement, that raises doubts about the party&apos;s ability to meet current or future obligations</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;DefaultEvent">
-		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;CreditEvent"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;ObligationSpecificCreditEvent"/>
 		<rdfs:label xml:lang="en">default event</rdfs:label>
 		<skos:definition xml:lang="en">credit event representing a failure to meet a contractual obligation, such as, but not limited to, failure to repay a debt including interest or principal on a loan or security</skos:definition>
```

Comment:
Should mention in notes that this allows demand of full amount and forfeiture of collateral. See https://www.investopedia.com/terms/e/event-of-default.asp#:~:text=An%20event%20of%20default%20is%20a%20pre%2Dspecified%20condition%20or,%2C%20or%20insolvency%2C%20among%20others.